examples/osgviewerIPhone/shaders.h

```diff
@@ -0,0 +1,113 @@
+#pragma once 
```

Comment:
looks like the travis build failed due to max time limit being reached. Is that something you can re trigger? can't see the changes I made making the build take any longer than usual